profiles/car.lua

```diff
@@ -471,9 +481,14 @@ function process_turn(profile, turn)
   local turn_bias = turn.is_left_hand_driving and 1. / profile.turn_bias or profile.turn_bias
 
   if turn.has_traffic_light then
```

Comment:
Initially I had a question "what to do if node has traffic light and stop sign at the same time?", IMO the answer is quite simple: here we just apply one of penalties, but end users may go further and apply combination of penalties in this case if they want or change their priority.

Reply:
I think applying the max penalty out of the three would make sense to me. This logic achieves that too.